plugins/grpc-transport-auth/src/main/java/com/github/bsideup/liiklus/transport/grpc/config/GRPCAuthConfiguration.java

```diff
@@ -39,40 +39,40 @@ public void initialize(GenericApplicationContext applicationContext) {
 
         log.info("GRPC Authorization ENABLED with algorithm {}", authProperties.getAlg());
 
-        // Init it early to check that everything is fine in config
-        JWTVerifier verifier = createVerifier(authProperties.getAlg(), authProperties);
-
         applicationContext.registerBean(
                 JWTAuthGRPCTransportConfigurer.class,
-                () -> new JWTAuthGRPCTransportConfigurer(verifier)
+                () -> new JWTAuthGRPCTransportConfigurer(authProperties)
```

Comment:
seems to be handled in #274, probably shouldn't be changed in this PR to avoid conflicts